kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -23,6 +23,7 @@
 import io.fabric8.kubernetes.client.internal.SSLUtils;
 import io.fabric8.kubernetes.client.utils.Utils;
 import io.sundr.builder.annotations.Buildable;
+import org.apache.commons.lang.StringUtils;
```

Comment:
We would like to avoid runtime dependency on commons-lang.

If I am not mistaken, its currently a transitive dependency of sundrio (which is compile time dependency). But no other code should directly use it.